projects/plugins/jetpack/extensions/blocks/subscriptions/class-jetpack-subscription-site.php

```diff
@@ -294,6 +294,8 @@ function ( $hooked_block, $hooked_block_type, $relative_position, $anchor_block
 </div>
 HTML;
 
+					$hooked_block['attrs']['appSource'] = 'subscribe-block-post-end';
+
```

Comment:
This works but I would move it where we call `get_post_end_placement_block_attributes`:
```php
$attrs = $this->get_post_end_placement_block_attributes(
	array(
		'style' => array(
			'spacing' => array(
				'margin'  => array(
					'top'    => '48px',
					'bottom' => '48px',
				),
				'padding' => array(
					'top'    => '5px',
					'bottom' => '5px',
				),
			),
		),
		'appSource' => 'subscribe-block-post-end'
	),
	$anchor_block
);
```
Up to you 😄

Reply:
I tried that first, but those attributes are for the `core/group` block.

Reply:
That's true, looks good then!